src/browser/components/MainPage.jsx

```diff
@@ -245,13 +251,33 @@ const MainPage = createReactClass({
     });
   },
 
-  focusOnWebView() {
-    this.refs[`mattermostView${this.state.key}`].focusOnWebView();
+  focusOnWebView(e) {
+    if (e.target.className !== 'finder-input') {
+      this.refs[`mattermostView${this.state.key}`].focusOnWebView();
+    }
+  },
+
+  activateFinder(state) {
```

Comment:
Please remove `state` to fix eslint error.

Reply:
Fixed it and pushed again.